### PR TITLE
Including pullapprove.yml file.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  pullapprove-admins:
+    conditions:
+      files:
+        - ".pullapprove.yml"
+    required: 1
+    users:
+      - iclanton
+      - nickpape-msft
+      - pgonzal
+      - qz2017
```
Comment (pgonzal) [inline .pullapprove.yml]: > qz2017 [](start = 7, length = 7)

Add Pat as an administrator

File: .pullapprove.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+version: 2
```
Comment (pgonzal) [inline .pullapprove.yml]: what are all these "^M" characters?

**Comment by pgonzal** [comment]
:clock1:

File: .pullapprove.yml (hunk 2 above)
Comment (iclanton) [inline .pullapprove.yml]: Invalid line endings. They're fixed now.

**Comment by iclanton** [comment]
Approved

**Comment by iclanton** [comment]
Approved

**Pull request merged**